pepper-apis/config/pepper-apis-settings.xml

```diff
@@ -1,18 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <profiles>
-        <profile>
-            <id>sonar</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <properties>
-                <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-                <sonar.projectName>ddp-study-server</sonar.projectName>
-                <sonar.projectKey>broadinstitute_ddp-study-server</sonar.projectKey>
-                <sonar.organization>dsp-appsec</sonar.organization>
-            </properties>
-        </profile>
-    </profiles>
+  <pluginGroups>
+    <pluginGroup>
+      org.sonarsource.scanner.maven
```

Comment:
What is this?

Reply:
`org.sonarsource.scanner.maven` is the SonarQube scanner that is integrates into a Maven build and runs on Java classfiles after compilation.